### ISSUE TITLE
Solid phase calculations missing?

### ISSUE THREAD

**Issue by Olaf987** [usnistgov/REFPROP-issues#627]
Dear all,

I have a simple question. It seems that REFPROP will not do calculations below the freezing point. I tried water and glycol. To me it appears that equations of state for the solid state have not been implemented at all. Is that correct? Or am I doing something wrong?

If the solid phase is not implemented, perhaps that should be made clear in the help file and the documentation.

Thanks for your help.

Olaf

**Comment by ianhbell** [comment]
That is correct. The EOS in REFPROP have no idea about the location of the melting curve. A note to this effect is one of the warnings in the GUI, but no note is made in the shared library itself.

**Comment by nist-aharvey** [comment]
A supplement to Ian's comment.
The **location** of the melting curve is implemented in REFPROP, at least for a number of fluids where that is available (such as water; I see we don't have one for ethylene glycol). So you can calculate (for a pure fluid) whether a point is in the solid or fluid phase by looking at the melting (or sublimation) curve for these fluids.
However as Ian mentions REFPROP does not calculate the properties of the solid phase. There are a few substances for which solid phase equations of state have been developed in the literature (such as H2O and CO2), but REFPROP only calculates properties for fluid phases (the full name suggests that restriction: "Reference **Fluid** Thermodynamic and Transport Properties" ).

**Comment by ianhbell** [comment]
I had interpreted your initial question as a _mixture_ of water and glycol (there are two glycols in REFPROP). There is no melting curve for glycol+water, but there is a reliable one for pure water (watch out for the multiple ice phases), and I think no melting curves for pure glycol.

**Comment by nist-aharvey** [comment]
Yes, if the question was for a mixture, even the location of the solid-phase boundary for a mixture (much less the properties of the solid) is in general a very difficult problem. There is a relatively straightforward approach to predicting the boundary (for which REFPROP could be used as part of the analysis) if you make the assumption that each pure component only precipitates as a pure solid with no mixed solid phases or solid compounds, but there are many mixtures where that assumption would be wrong.  Solid formation from mixtures requires analysis for each specific system and could not be done in any sort of reliable way in the context of a general fluid mixture package.

**Comment by Olaf987** [comment]
Dear all:

Thanks for the answers. I realize melting curves of mixtures are a nightmare. Being a first-time REFPROP user the first thing I did was trying water from -20 to 200C. I was expecting that I would see a solid phase appearing below 0C. But that did not happen. It more or less gave up below 0C. I will try a melting curve Monday.

For now I am just happy just to know I did not make a mistake.

Olaf
